docs/user/migration/external-name-alias.md

```diff
@@ -0,0 +1,148 @@
+# Migration Guide for External Name Alias's Behavior Change
+
+## Context
+
+Istio with 1.21 version has changed the behavior of the Service of type ExternalName.
+Previously, for the Service of type ExternalName a separate `cluster` in the Envoy config was created.
+With the change, the Service of type ExternalName is treated as an alias of the Service that it points to.
+Istio introduced this change to align the way it handles the Service of type ExternalName with the Kubernetes behavior.
+Since it caused some issues for our users, we introduced a new annotation, **disable-external-name-alias**, to disable this behavior.
+If you are affected by this change, update your configuration to align with the new behavior.
```

Comment:
```suggestion
Since it caused some issues for our users, we introduced a new annotation, **disable-external-name-alias**, to disable this change.
However, use of this parameter is not recommended, and Istio provides a migration path from the previous behaviour.
If you are using this parameter, please update your configuration according to the following points.
```

Reply:
This way, I think we are encountering a problem that the Istio migration guide from 1.21 upgrade notes does not fully work. I think mentioning that might be harmful. Someone might go there and use it.